Packages.Data.props

```diff
@@ -34,9 +34,9 @@
   <ItemGroup>
     <PackageReference Update="NUnit" Version="3.13.2" />
     <PackageReference Update="System.ClientModel" Version="1.0.0-beta.3" />
-    <PackageReference Update="Azure.Core" Version="1.36.0" />
+    <PackageReference Update="Azure.Core" Version="1.37.0" />
     <PackageReference Update="Azure.Core.Experimental" Version="0.1.0-preview.18" />
-    <PackageReference Update="Azure.ResourceManager" Version="1.9.0" />
+    <PackageReference Update="Azure.ResourceManager" Version="1.11.0-alpha.20240210.4" />
```

Comment:
This causes the GA System.ClientModel to be pulled in which has many breaking changes from the beta.

Reply:
Lets merge this into a feature branch `feature/cdk` for now until we can work out releasing a beta resourcemanager with this new MRWO type.